src/measure_theory/constructions/prod.lean

```diff
@@ -656,8 +656,8 @@ by { simp_rw [← lintegral_prod_swap f hf], exact lintegral_prod _ hf.prod_swap
 /-- The symmetric verion of Tonelli's Theorem: For `ℝ≥0∞`-valued measurable
 functions on `α × β`,  the integral of `f` is equal to the iterated integral, in reverse order. -/
 lemma lintegral_prod_symm [sigma_finite μ] (f : α × β → ℝ≥0∞)
-  (hf : ae_measurable f (μ.prod ν)) : ∫⁻ z, f z ∂(μ.prod ν) = ∫⁻ y, ∫⁻ x, f (x, y) ∂μ ∂ν :=
-lintegral_prod_symm' f hf
+  (hf : measurable f) : ∫⁻ z, f z ∂(μ.prod ν) = ∫⁻ y, ∫⁻ x, f (x, y) ∂μ ∂ν :=
+lintegral_prod_symm' f hf.ae_measurable
```

Comment:
This correction is correct. I apparently messed this up in #5948

Reply:
On the other hand, this is now inconsistent with `lintegral_prod`. Maybe a better fix (instead of this one) is to remove `lintegral_prod_symm'`.

Reply:
Isn't the point that the primed version has weaker assumptions?

Reply:
not *before* this change. I'm basically suggesting we remove the version with stronger assumptions.

Reply:
I'm generally a fan of leaving the stronger assumption versions, just for discoverability by library_search if nothing else (I've no idea if library search would find the ae version from measurable assumptions without).

Reply:
> On the other hand, this is now inconsistent with `lintegral_prod`. Maybe a better fix (instead of this one) is to remove `lintegral_prod_symm'`.

@alexjbest What is your proposed solution?

Reply:
> I'm generally a fan of leaving the stronger assumption versions, just for discoverability by library_search if nothing else (I've no idea if library search would find the ae version from measurable assumptions without).

That's also fine by me. Could you then use the same pattern for `lintegral_prod` (a weaker assumption `lintegral_prod'` and stronger assumption `lintegral_prod`)?

Reply:
Ok I've swapped the declarations lintegral_prod_symm and the primed version